examples/with-persistent/src/Main.hs

```diff
@@ -0,0 +1,43 @@
+{-# language FlexibleContexts      #-}
+{-# language OverloadedStrings     #-}
+{-# language PartialTypeSignatures #-}
+{-# language TypeApplications      #-}
+{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+
+module Main where
+
+import           Control.Monad.IO.Class  (liftIO)
+import           Control.Monad.Logger
+import           Data.Conduit
+import           Data.Pool
+import           Database.Persist.Sqlite
+import           Mu.GRpc.Server
+import           Mu.Server
+
+import           Schema
+
+main :: IO ()
+main = do
+  putStrLn "running application with persistent"
+  runMigration migrateAll
+  runNoLoggingT $
+    withSqlitePool @(NoLoggingT IO) @SqlBackend ":memory:" 10 $ \pool ->
+      runGRpcApp 1234 server pool
+
+server :: MonadServer m => Pool SqlBackend -> ServerT PersistentService m _
+server p = Server (getPerson p :<|>: newPerson p :<|>: allPeople p :<|>: H0)
+
+getPerson :: Monad m => Pool SqlBackend -> PersonRequest -> m Person
+getPerson pool (PersonRequest name) = liftIO $ flip runSqlPersistMPool pool $ do
+  p <- getBy $ UniquePerson name
```

Comment:
Like in the case of `STM` blocks, it's important to bound the amount of code run in a `runSqlPersistMPool` block, because during that time we are taking the connection for our use. In this case the block is small enough, but be aware of that fact.